staging/src/k8s.io/api/authorization/v1/types.go

```diff
@@ -171,6 +171,8 @@ type SelfSubjectAccessReviewSpec struct {
 type SubjectAccessReviewStatus struct {
 	// Allowed is required.  True if the action would be allowed, false otherwise.
 	Allowed bool `json:"allowed" protobuf:"varint,1,opt,name=allowed"`
+	// Denied is required.  True if the action would be denied, false otherwise.
+	Denied bool `json:"denied" protobuf:"varint,4,opt,name=denied"`
```

Comment:
`omitempty`, `+optional`, since it's a new field, and the description could use expanding. May not be true if Allowed is true.